src/sc/src/k8/objects/spg_group.rs

```diff
@@ -217,16 +220,29 @@ mod k8_convert {
                 ..Default::default()
             });
 
+            spu_non_tls_public = 9007;
+
             args.push("--bind-non-tls-public".to_owned());
-            args.push("0.0.0.0:9007".to_owned());
+            args.push(format!("0.0.0.0:{}", spu_non_tls_public));
         }
 
         // add RUST LOG, if passed
         if let Ok(rust_log) = std::env::var("RUST_LOG") {
             env.push(Env::key_value("RUST_LOG", &rust_log));
         }
 
-        // env.append(&mut spu_template.env.clone());
+        let liveness_probe = Probe {
```

Comment:
Don't think liveness probes should run in the SC

Reply:
Here it is being added to the SPG StatefulSet config

Reply:
This just allows the SC to configure the K8 probe right? So K8 will be running the probe but the SC is the thing that sets it up in the  cluster? If I am interpreting the flow right

Reply:
@nicholastmosher Correct